### ISSUE TITLE
Disabling notifications 

### ISSUE THREAD

**Issue by Marin-b** [orchidsoftware/platform#2000]
**Describe the bug**
Trying to add Orchid on top of an already large project. There is no need for notifications so there is no `notifications` table and no `Notifiable` in our AdminUser Model.
I disabled them in the platform.php config file but there is still an error coming from the constructor of the [Notification](https://github.com/orchidsoftware/platform/blob/master/src/Platform/Components/Notification.php) component:
```
Call to undefined method AdminUser::unreadNotifications()
```

**To Reproduce**

  1. Remove `notifiable` from User model
  2. Disable notification in config file:
    ```
     'notifications' => [
        'enabled'  => false,
        'interval' => 60,
    ]
    ```
  3. Try to load any page of the dashboard

**Expected behavior**
To not raise an error about missing unreadNotification method. 

I think an easy solution would be to move the unreadNotification query outside of the [Notification](https://github.com/orchidsoftware/platform/blob/master/src/Platform/Components/Notification.php) constructor into a member method. 
The rest should be handled with the `shouldRender()`

**Comment by tabuna** [comment]
Should be fixed with release 10.25.3

**Comment by Marin-b** [comment]
Perfect, thank you!
